mex/backend/graph/cypher/fetch_merged_items.cql

```diff
@@ -21,6 +23,11 @@ Returns:
 -#>
 CALL () {
 <%- block match_clause -%>
+<%- if filter_by_reference_to_merged_item %>
```

Comment:
this needs to go on the same nesting level as `filter_by_identifier`,
otherwise it'll crash when combining `hadPrimarySource` with a `q` search filter.

`CypherSyntaxError: {code: Neo.ClientError.Statement.SyntaxError} {message: Variable merged_node already declared in outer scope...}`